interface/billing/billing_report.php

```diff
@@ -1142,14 +1144,15 @@ function criteriaSelectHasValue(select) {
                                     $last_level_closed = sqlQuery("SELECT `last_level_closed` FROM `form_encounter` WHERE `encounter` = ?", array($iter['enc_encounter']))['last_level_closed'];
                                     $effective_insurances = getEffectiveInsurances($iter['pid'], $iter['enc_date']);
```

Comment:
hi @juggernautsei, maybe we should fix the return in the function `getEffectiveInsurances` to return an empty array, 
https://github.com/openemr/openemr/blob/master/library/patient.inc.php#L1566
```
    return $insarr ?? [];

Reply:
@stephenwaite the function already returns an empty array. Adding the null collacent would insure an empty array is returned but not necessary because the value is initialized as an empty array. But we can add it as to make double sure an empty array is returned.